h2o-core/src/main/java/hex/grid/Grid.java

```diff
@@ -540,6 +535,68 @@ public void exportModelsBinary(final String exportDir) throws IOException {
     }
   }
 
+  @SuppressWarnings("rawtypes")
+  private Set<Key> getDependentKeysFromParams() {
+    Field[] fields = Weaver.getWovenFields(getParams().getClass());
```

Comment:
Should we instead use ModelParameterSchema class to get fields (and possibly their metadata)?